meson.build

```diff
@@ -342,7 +342,7 @@ if host_machine.system() == 'darwin'
 
   # Locale discovery
   corefoundation_dep = dependency('appleframeworks',
-                                  modules : ['CoreFoundation'],
+                                  modules : ['CoreFoundation', 'CoreServices'],
                                   required : false)
   if corefoundation_dep.found()
     if cxx.check_header('CoreFoundation/CoreFoundation.h')
```

Comment:
CoreFoundation is needed for `locale` handling. If we bundle SDL-CD's dependencies into this check, then it becomes over-prescribed: because now if _either_ check fails, we won't get locale or SDL-CD. 

Suggest making its own dep:

``` python
  # SDL CD dependency
  coreservices_dep = dependency('appleframeworks',
                                modules : ['CoreServices'],
                                required : false)  
  if coreservices_dep.found()
    if cxx.check_header('CoreServices/CoreServices.h')
      conf_data.set('C_CORESERVICES', 1)
    else
      warning('''Core Services support disabled: header is unsable''')
    endif
  else
    warning('''Core Services support disabled: compiler can't detect/use Apple Frameworks''')
  endif
```

And in `src/config.h.in`, insert this along side the other Apple C_CORE:

```c
// Compiler supports Core Services headers
#mesondefine C_CORESERVICES
```

Reply:
Thanks for the suggestion, which certainly improves the build system. Done.